### PR TITLE
Remove regex pattern for ReturnUrl

### PR DESCRIPTION
**Pull request by OFollan**
Also, RedirectUrl is not always present (e.g. for PUSH_MESSAGE userflow)

### CODE REVIEW

**Review by tht13 — COMMENTED** [review]
Can we add something to the description field that specifies what schemas are not allowed?

We also need to disallow IP addresses and some other security things, so this should be clearly documented for the user

**Comment by OFollan** [comment]
>  The URL the user is redirected to after a successful payment. Has to be either `https://` or a custom URL scheme.

Is this understandable enough?

File: docs/epayment.v1.yml [hunk 1]
```diff
@@ -274,6 +274,12 @@ components:
         returnUrl:
           type: string
           example: 'https://example.com/redirect?orderId=acme-shop-123-order123abc'
+          description: |-
+            The URL the user is redirected to after the payment session.
+            
+            If Userflow is `WEB_REDIRECT` the URL has to be `https://`.
+            
+            If Userflow is `NATIVE_REDIRECT` the URL has to be `https://` or a custom URL scheme.
```
Comment (tht13) [inline docs/epayment.v1.yml:282]:
```suggestion
            The URL the user is returned to after the payment session.
            
            The URL has to use the `https://` scheme or a custom URL scheme.
```

Comment (tht13) [reply docs/epayment.v1.yml:282]: Web flow should allow a custom scheme return url

**Review by tht13 — APPROVED** [review]

**Pull request merged**